prometheus_client/core.py

```diff
@@ -198,6 +198,11 @@ def add_metric(self, labels, value):
           labels: A list of label values
           value: The value of the metric.
         '''
+        try:
```

Comment:
I reviewed the string_types definition in the **six** project.
[GitHub-benjaminp/six](https://github.com/benjaminp/six/blob/master/six.py)
``` python
if PY3:
    string_types = str,
    integer_types = int,
    class_types = type,
    text_type = str
    binary_type = bytes

    MAXSIZE = sys.maxsize
else:
    string_types = basestring,
    integer_types = (int, long)
    class_types = (type, types.ClassType)
    text_type = unicode
    binary_type = str
```